external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/DockerJDBCIntegrationSuite.scala

```diff
@@ -62,10 +62,17 @@ abstract class DatabaseOnDocker {
    * Optional process to run when container starts
    */
   def getStartupProcessName: Option[String]
+
+  /**
+   * Optional step before container starts
+   */
+  def beforeContainerStart(hostConfigBuilder: HostConfig.Builder,
+      containerConfigBuilder: ContainerConfig.Builder): Unit = {}
```

Comment:
nit. Could you put the first argument on the next line?
```scala
def beforeContainerStart(
    hostConfigBuilder: HostConfig.Builder,
    containerConfigBuilder: ContainerConfig.Builder): Unit = {}
```

Reply:
Fixed.